backend/index.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+const pool = require("./db");
+const PORT = process.env.PORT || 3001;
+
+//middleware
+
+app.use(cors());
+app.use(express.json());
+
+
+
+
+
+app.post("/sign-in", async (req, res) => {
+  try {
+    const { firstName, lastName, login, password } = req.body;
+    const newTodo = await pool.query("INSERT INTO todo (firstName, lastName, login, password) VALUES($1, $2, $3, $4) RETURNING *", [firstName, lastName, login, password]);
+    res.json(newTodo.rows[0]);
+
+  } catch (err) {
+    console.error(err.message);
+  }
+});
+
+app.get("/sign-in", async (req, res) => {
+  try {
+    const allTodo = await pool.query("SELECT * FROM todo");
+    res.json(allTodo.rows);
+  } catch (err) {
+    console.error(err.message);
+    
+  }
+});
```

Comment:
this route doesn't make sense to me, what should it do?

Reply:
This route list all users that are in database, it's necessary?

Reply:
don't think so. It's not in the doc you created.

Reply:
just remove this one and you can merge

Reply:
ok

Reply:
wait, so I have one question about the new route with bearer, Can you review?